website/server/controllers/api-v3/groups.js

```diff
@@ -1066,6 +1067,31 @@ api.inviteToGroup = {
     let uuids = req.body.uuids;
     let emails = req.body.emails;
 
+    // If party, check the limit of members
+    if (group.type === 'party') {
+      let partyLimitMembers = 5;
```

Comment:
actually since the limit is used in the jade file below it should probably be put in common/script/constants so that it can be used in both the server and the client